pipelines/migration/projeto_subsidio_sppo/constants.py

```diff
@@ -709,3 +710,289 @@ class constants(Enum):  # pylint: disable=c0103
+            },
+            "dbt_utils.accepted_range__viagens_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna viagens_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__viagens_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna viagens_faixa maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_planejada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_planejada_faixa maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_apurada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_apurada_faixa maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_subsidiada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_subsidiada_faixa maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__pof__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna pof maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_apurada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_apurada_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_subsidiada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_subsidiada_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__km_planejada_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_planejada_dia maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__media_pof__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna media_pof maiores ou igual a zero"
+            },
+            "dbt_utils.accepted_range__valor_a_pagar__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna valor_a_pagar maiores ou igual a zero"
+            },
+            "dbt_utils.unique_combination_of_columns__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as linhas de sumario_faixa_servico_dia_pagamento são unicas"
+            },
+            "dbt_expectations.expect_row_values_to_have_data_for_every_n_datepart__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as datas possuem dados"
+            },
+            "check_km_planejada__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todas as viagens possuem km_planejada correspondente a OS"
```

Comment:
```suggestion
                "description": "Todas as viagens possuem `km_planejada` correspondente à OS"
```
Alterar para todos os casos semelhantes